Sources/SharedServices/DynamoUserRepository.swift

```diff
@@ -84,39 +94,31 @@ struct DynamoUserRepository {
     }
     
     // MARK: - Link users
-    func linkGitHub(with discordId: String, _ githubId: String) async throws -> String {
-        // TODO: Implement
-        // Check if the users github already exists
-        
-        // If it exists, merge the 2 accounts
-        
-        // If the the given discordId already has a github account linked, overwrite the githubId
-        
-        // Delete the account that's not up-to-date
-        abort()
-    }
-    
-    func linkDiscord(with githubId: String, _ discordId: String) async throws -> String {
-        // TODO: Implement
-        // Check if the users discord already exists
-        
-        // If it exists, merge the 2 accounts
-        
-        // If the the given discordId already has a github account linked, overwrite the githubId
-        
-        // Delete the account that's not up-to-date
-        abort()
-    }
-    
-    private func mergeAccounts() async throws -> Bool {
-        // TODO: Implement
-        // Return true if the merge was successful
-        abort()
-    }
-    
-    private func deleteAccount() async throws -> Bool {
-        // TODO: Implement
-        // Return true if the deletion was successful
-        abort()
+
+    func linkGithubID(to discordID: String, _ githubID: String) async throws {
```

Comment:
just do `linkGithubID(discordID: String, githubID: String)` so it's clear at call sites what is exactly happening and what the developer is supposed to pass to the function, and where.